src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -576,6 +576,9 @@
     <Field Name="GetAwaiter" Type="BoundExpression?" Null="allow"/>
     <Field Name="IsCompleted" Type="PropertySymbol?" Null="allow"/>
     <Field Name="GetResult" Type="MethodSymbol?" Null="allow"/>
+
+    <!-- This should be removed. https://github.com/dotnet/roslyn/issues/49218 -->
+    <Field Name="BinderOpt" Type="Binder?" Null="allow"/>
```

Comment:
Sorry, I thought this would enable me to avoid creating binders during lowering, but that didn't work out and I neglected to revert the change.

I would like to either:
1. as a follow-up, push more creation of calls that may contain default arguments into binding. Things such as Dispose and GetAwaiter calls on async enumerables, or a Dispose call on a foreach.
2. modify BindDefaultArgument to not depend on `GenerateImplicitConversionError` and `CreateConversion`, then make the method static so it can be used from lowering. They were just so darn convenient..